### PR TITLE
Bxmsprod 1026

### PR DESCRIPTION
**Pull request by akoufoudakis**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/bacon/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/rhba/OfflineManifestGenerator.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        for (PncBuild build : builds.values()) {
+            artifactsToListRaw.addAll(build.getBuiltArtifacts());
+            // TODO: Add filter, basing on the targetRepository.repositoryType, when NCL-6079 is done
+            buildInfoCollector.addDependencies(build, "");
+            artifactsToListRaw.addAll(build.getDependencyArtifacts());
+            log.debug("Dependencies for build {}: {}", build.getId(), build.getDependencyArtifacts().size());
+        }
+        log.debug("Number of collected artifacts for the Offline manifest: {}", artifactsToListRaw.size());
+
+        List<String> exclusions = pigConfiguration.getFlow().getRepositoryGeneration().getExcludeArtifacts();
```
Comment (michalszynkiewicz) [inline pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/rhba/OfflineManifestGenerator.java:63]: is `excludeArtifacts` sth that was added for offliner or was it there before?

**Comment by michalszynkiewicz** [comment]
Could you add a test for it?

File: pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/rhba/OfflineManifestGenerator.java (hunk 1 above)
Comment (akoufoudakis) [reply pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/rhba/OfflineManifestGenerator.java:63]: It's already there in the repo generation section.

**Comment by michalszynkiewicz** [comment]
Thanks for adding the test!

**Comment by thescouser89** [comment]
Please only merge PRs when we have a successful CI run. This time it failed due to a simple formatting rule that failed, but that's causing all the new PRs created to fail as a result.

Thanks! :)

**Comment by michalszynkiewicz** [comment]
oops, I didn't notice the failure, sorry

**Comment by thescouser89** [comment]
No worries! :)

**Pull request merged**